packages/terra-functional-testing/src/config/capabilities.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This file contains the capability configurations for each supported browser.
+ */
+
+module.exports = {
+  chrome: {
```

Comment:
Do we no longer need to set [loggingPrefs](https://github.com/cerner/terra-toolkit-boneyard/blob/ea4a5fc159f8e5f71911077ef613259694940e47/config/wdio/selenium.config.js#L6-L8) and [perfLoggingPrefs](https://github.com/cerner/terra-toolkit-boneyard/blob/ea4a5fc159f8e5f71911077ef613259694940e47/config/wdio/selenium.config.js#L15-L17)? I don't know who looks at the performance logs or what they are used for.

Reply:
I left a short description in the PR summary about the logging. Some of the APIs have changed/been removed and I wasn't able to find any concrete documentation on how the logging was actually intended to be used. I haven't seen any examples of people using the logging, so I think this is something we can passively add back in later if necessary.

Here's a few related issues I stumbled across while looking around:

https://github.com/webdriverio/webdriverio/issues/590 
https://github.com/webdriverio/webdriverio/issues/476
http://v4.webdriver.io/guide/testrunner/browserobject.html

Reply:
Oops, I didn't pay attention to your details on the PR summary.